enclave_ops/deployment/dkeycache/Enclave/enclave_ra.cpp

```diff
@@ -236,6 +237,27 @@ sgx_status_t enclave_init_ra(
 {
     // isv enclave call to trusted key exchange library.
     sgx_status_t ret;
+
+    std::string pub = PUB_RESULT;
+    char* buffer = strndup(pub.c_str(), pub.length() + 1);
+    /* Skip the 1st byte which indicates uncompressed form */
+    char* pch = strtok(buffer, ":");
+    uint32_t i = 0;
+
+    while (pch != NULL && i < sizeof(sgx_ec256_public_t)) {
+        pch = strtok(NULL, ":");
+        uint8_t value = std::stoi(std::string(pch), 0, 16);
+
+        /* Reversely set 64B public key gx/gy due to little endian sgx_ec256_public_t */
+        if (i < SGX_ECP256_KEY_SIZE)
+            g_sp_pub_key.gx[SGX_ECP256_KEY_SIZE - 1 - i] = value;
+        else
+            g_sp_pub_key.gy[2 * SGX_ECP256_KEY_SIZE - 1 - i] = value;
+        i++;
+    }
+
+    free(buffer);
+
```

Comment:
suggest putting the above codes into a separate function: 
initialize_sp_pub_key()?